docs/release-notes.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Release Notes
+---
+
+## v2.0.0
+
+### New Features
+
+- Support for [LoRa Basics™ Station](https://github.com/lorabasics/basicstation)
+LNS Protocol (v2.0.5)
+- Support for [LoRa Basics™ Station](https://github.com/lorabasics/basicstation)
+CUPS Protocol (v2.0.5) for credential management and firmware upgrade
+- LoRaWAN Network Server and Basics™ Station can be decoupled, allowing the
+possibility to connect multiple concentrators on a single LoRaWAN Network Server
+instance
+- Support for secure communication between LoRaWAN Network Server and Basics™
+Station
+- Support for running LoRaWAN Network Server on [Azure IoT Edge for Linux on
+Windows](https://docs.microsoft.com/en-us/azure/iot-edge/iot-edge-for-linux-on-windows?view=iotedge-2018-06)
+- Regional support for AS923 and CN470 (Regional Parameters v1.0.3rA and
+RP002-1.0.3 revisions)
+- Support for running LNS on a separate Edge or Cloud device
+- [Observability](./user-guide/observability.md):
+  - Integration with Azure Monitor
+  - Exposing a Prometheus endpoint for metrics scraping
+  - Collection of metrics for edgeAgent and edgeHub modules with [Metrics collector module](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-collect-and-transport-metrics?view=iotedge-2020-11&tabs=iothub)
+  - distributed tracing
+
+### Code Quality Improvements
+
+- Introduction of LoRaWAN Primitives and refactoring of existing code to make
+wide use of those
+- Upgrade to .NET 6 (LTS version)
+- Upgrade container base images to Debian 11
+- Improvements in E2E CI, continuously testing multiple scenarios and
+authentication modes
+
+### Breaking Changes
+
+- Deprecated support for Packetforwarder ([upgrade instructions](user-guide/pkt-fwd-to-station.md))
+- Upgrade to RaspberryPi OS based on Debian 11.0 Bullseye ([upgrade instructions](user-guide/upgrade.md#upgrading-to-raspberry-pi-os-bullseye))
+- Infrastructure changes:
```

Comment:
Let's add that the Azure Function extension version needs to be updated to ~4 to support v2.0.0

Reply:
maybe link the upgrade instructions in general rather than repeating that information: https://azure.github.io/iotedge-lorawan-starterkit/dev/user-guide/upgrade/